src/main/java/taxi/service/impl/RegistrationServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package taxi.service.impl;
+
+import taxi.dao.DriverDao;
+import taxi.exception.RegistrationException;
+import taxi.lib.Inject;
+import taxi.lib.Service;
+import taxi.model.Driver;
+import taxi.service.RegistrationService;
+
+@Service
+public class RegistrationServiceImpl implements RegistrationService {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver register(Driver driver, String repeatedPassword) throws RegistrationException {
+        if (driverDao.findByLogin(driver.getLogin()).isPresent()
+                || !driver.getPassword().equals(repeatedPassword)) {
+            throw new RegistrationException("Driver is already exists");
+        }
+        return driverDao.create(driver);
```

Comment:
![image](https://user-images.githubusercontent.com/117590814/215402772-cd80ddc2-a18a-404d-9132-fae9e9420307.png)
there was no need to write logic here with one if. you need to throw two different exceptions here
- when passwords equals
- when login is already present in db